docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+
+services:
+  mongo:
+    container_name: mongo
+    image: mongo
+    volumes:
+      - ./data:/data/db
+    ports:
+      - "27017:27017"
+  serverless:
+    build: .
+    depends_on:
+      - mongo
+    ports:
+      - "8000:8000"
+    volumes:
```

Comment:
Could be wrong but this might not be necessary for the Mongo container to function (and thus no reason to worry about mounting the data folder).

Reply:
Also; we should update the `Makefile` to run `docker-compose up` now

Reply:
I think the `volumes:` thing is useful for persisting the volume between runs -- which would be useful if ever ran the endpoint locally -- and indeed not useful at all now. Hm yeah so I'll delete it.

Reply:
Hm actually I'll leave it in.